dockernet/README.md

```diff
@@ -89,11 +96,11 @@ chains:
     value:
       key: rly{N}
       chain-id: {CHAIN_ID}
-      rpc-addr: http://{NODE_PREFIX}1:26657
-      account-prefix: {ACCOUNT_PREFIX}
+      rpc-addr: http://{node_prefix}1:26657
```

Comment:
Maybe actually just change it to `{new-host-zone}` so then it serves both purposes of being clear what it represents, and being lowercase to avoid confusion

Reply:
done